### ISSUE TITLE
Support of Angular 9

### ISSUE THREAD

**Issue by stephanie-b1n3t** [leovo2708/ngx-treeview#254]
Hello, 

Many thanks for this module who worked very well till upgrading towards Angular 9. 
Is it possible to implement support of Angular 9 ? 
Regards :)

Version : 6.0.2

```
ERROR TypeError: Cannot read property 'selection' of undefined
    at DropdownTreeviewComponent.push../node_modules/ngx-treeview/__ivy_ngcc__/src/dropdown-treeview.component.js.DropdownTreeviewComponent.getText (dropdown-treeview.component.js:22)
    at DropdownTreeviewComponent_Template (dropdown-treeview.component.js:72)
```

**Comment by stephanie-b1n3t** [comment]
Hi :) !
Some news about support of Angular 9 ? 
Many thanks  :)

**Comment by brijeshpant83** [comment]
its giving me error too on angular 9

Uncaught (in promise): Error: Template parse errors: Can't bind to 'config' since it isn't a known property of 'ngx-treeview'.

**Comment by Darius123** [comment]
I am considering using this library. Would love it if it worked with angular 9 and if it was easy to decouple it from bootstrap 4 since I am planning to use angular material

**Comment by ingenmaffen** [comment]
I was having the same issue so I fixed it myself: https://www.npmjs.com/package/@ingenmaffen/ngx-treeview
Please consider that the only working version is 9.1.1 (this is the first time I published an npm package and I had some complications with it).

**Comment by Ednilso** [comment]
Please,

What's wrong with applying the command

npm i @ingenmaffen/ngx-treeview

Error:

0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'install',
1 verbose cli   '@ingenmaffen/ngx-treeviewnpm',
1 verbose cli   '--save'
1 verbose cli ]
2 info using npm@6.12.0
3 info using node@v12.13.0
4 verbose npm-session 005beef2365b0ccf
5 silly install loadCurrentTree
6 silly install readLocalPackageData
7 http fetch GET 404 https://registry.npmjs.org/@ingenmaffen%2fngx-treeviewnpm 492ms
8 http fetch GET 404 https://registry.npmjs.org/@ingenmaffen%2fngx-treeviewnpm 46ms
9 silly fetchPackageMetaData error for @ingenmaffen/ngx-treeviewnpm@latest 404 Not Found - GET https://registry.npmjs.org/@ingenmaffen%2fngx-treeviewnpm - Not found
10 timing stage:rollbackFailedOptional Completed in 1ms
11 timing stage:runTopLevelLifecycles Completed in 2958ms
12 verbose stack Error: 404 Not Found - GET https://registry.npmjs.org/@ingenmaffen%2fngx-treeviewnpm - Not found
...
13 verbose statusCode 404
14 verbose pkgid @ingenmaffen/ngx-treeviewnpm@latest
...
20 error code E404
21 error 404 Not Found - GET https://registry.npmjs.org/@ingenmaffen%2fngx-treeviewnpm - Not found
22 error 404
23 error 404 '@ingenmaffen/ngx-treeviewnpm@latest' is not in the npm registry.
24 error 404 You should bug the author to publish it (or use the name yourself!)
25 error 404 Note that you can also install from a
26 error 404 tarball, folder, http url, or git url.
27 verbose exit [ 1, true ]

Att,

Edniso
ednilso@hotmail.com

**Comment by YogeshBorkhade** [comment]
does anyone found any solution?

**Comment by NitinJain27** [comment]
Any update on the fix of this

**Comment by leovo2708** [comment]
Please try the latest version, I think it is compatible with angular 9

**Comment by tanmayjoshi1412** [comment]
Any fix for this? Facing same issue.
Angular Version : 9

**Comment by scorp559** [comment]
i am facing same


` error TS2305: Module '"../../../../../../../../../../../Development/Sprint/Ng-Changes-9/RestaurantIntranet/RestaurantIntranet.WebUI/ClientApp/node_modules/ngx-treeview/ngx-treeview"' has no exported member 'TreeviewI18nDefault'.

1 import { TreeviewItem, TreeviewSelection, TreeviewI18nDefault } from 'ngx-treeview';`

**Comment by meenalgupta23** [comment]
Facing exactly the same issue. Any fix for it?

**Comment by scorp559** [comment]
yes it is resolved : i am using angular version 10.1.6

1.  updated  "ngx-treeview": "^10.0.2", from  "ngx-treeview": "^6.0.2",
2.
`import { TreeviewItem, TreeviewSelection, DefaultTreeviewI18n } from 'ngx-treeview';

export class DropdownTreeviewSelectI18n extends DefaultTreeviewI18n {}`

`  @ViewChild('dropdownMenu', { static: true })` to `  @ViewChild('dropdownMenu', { static: false })`


and also commented code
`private selectItem(item: TreeviewItem) {
    this.dropdownTreeviewComponent.dropdownDirective.close(); // need to comment this
    if (this.dropdownTreeviewSelectI18n.selectedItem !== item) {
      this.dropdownTreeviewSelectI18n.selectedItem = item;
      if (this.value !== item.value) {
        this.value = item.value;
        this.valueChange.emit(item);
      }
    }
  }`

**Comment by scorp559** [comment]
> Facing exactly the same issue. Any fix for it?

yes it is resolved : i am using angular version 10.1.6

updated "ngx-treeview": "^10.0.2", from "ngx-treeview": "^6.0.2",
`import { TreeviewItem, TreeviewSelection, DefaultTreeviewI18n } from 'ngx-treeview';

export class DropdownTreeviewSelectI18n extends DefaultTreeviewI18n {}`

  @ViewChild('dropdownMenu', { static: true }) to   @ViewChild('dropdownMenu', { static: false })

and also commented code
private selectItem(item: TreeviewItem) { this.dropdownTreeviewComponent.dropdownDirective.close(); // need to comment this if (this.dropdownTreeviewSelectI18n.selectedItem !== item) { this.dropdownTreeviewSelectI18n.selectedItem = item; if (this.value !== item.value) { this.value = item.value; this.valueChange.emit(item); } } }

**Comment by meenalgupta23** [comment]
> yes it is resolved : i am using angular version 10.1.6


I need it for Angular version 9.1.13

**Comment by scorp559** [comment]
> > yes it is resolved : i am using angular version 10.1.6
> 
> I need it for Angular version 9.1.13

yes its for same too

**Comment by meenalgupta23** [comment]
> yes its for same too

`npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@"^10.0.0" from ngx-treeview@10.0.2`

ngx-treeview >=10.0.0 works with angular 10
